src/eduid_webapp/support/views.py

```diff
@@ -30,39 +32,29 @@ def index(support_user):
                 return render_template('index.html', support_user=support_user,
                                        error="No users matched the search query")
 
-        current_app.logger.info('Support personnel: {!r} searched for {!r}'.format(support_user, search_query))
+        current_app.logger.info('Support personnel {} searched for {}'.format(support_user, search_query))
         for user in lookup_users:
             user_data = dict()
-            # Users
+            user_dict = user.to_dict()
```

Comment:
This would have been a better place for the comment I already made elsewhere:

I think you should let userdb parse data into a User() of some sort and then build your dict from the User object. The dict representations of users is subject to change.